catkit/hardware/boston/BostonDmController.py

```diff
@@ -32,6 +31,8 @@ def initialize(self, serial_number, command_length, dac_bit_width):
         # Create class attributes for storing individual DM commands.
         self.dm1_command = None
         self.dm2_command = None
+        self.dm1_command_object = None
+        self.dm2_command_object = None
```

Comment:
These aren't actually referenced, but I added them here for possible future use (they could equally be nuked 🤷 ).